core/ResponseDefinitions.php

```diff
@@ -176,6 +176,34 @@
  *       iconURL: ?string,
  *       iconEmoji: ?string,
  *   }
+ *
+ * @psalm-type CoreTaskProcessingShape = array{
+ *     name: string,
+ *     description: string,
+ *     type: int,
+ *     mandatory: bool,
+ * }
+ *
+ * @psalm-type CoreTaskProcessingTaskType = array{
+ *     name: string,
+ *     description: string,
+ *     inputShape: CoreTaskProcessingShape[],
+ *     outputShape: CoreTaskProcessingShape[],
```

Comment:
```suggestion
 *     inputShapes: CoreTaskProcessingShape[],
 *     outputShapes: CoreTaskProcessingShape[],
```
Since they are an array the names should be plural as well.

Reply:
I disagree. It's the shape of the input. It's one input (because it's input once) so one shape.

Reply:
Hm that makes sense, how about just `input` and `output`?

Reply:
Well it's not an input but just the shape of the input to be :)